utils_test.py

```diff
@@ -188,3 +190,12 @@ def test_get_thumbnail_icon_url_for_category(self):
         self.assertEqual(
             utils.get_thumbnail_icon_url_for_category('Nonexistent'),
             '/images/subjects/Lightbulb.svg')
+
+    def test_parse_date_from_string(self):
+        self.assertEqual(
+            utils.parse_date_from_string(utils.get_current_date_as_string()),
```

Comment:
There are examples elsewhere in the codebase for testing errors, like "with self.assertRaisesRegexp". You can grep for these in order to get a sense of how they work.